maskrcnn_benchmark/config/defaults.py

```diff
@@ -54,6 +54,12 @@
 # Convert image to BGR format (for Caffe2 models), in range 0-255
 _C.INPUT.TO_BGR255 = True
 
+# Image ColorJitter
+_C.INPUT.BRIGHTNETSS = 0.0
```

Comment:
typo: BRIGHTNESS

Reply:
Hi, 

I have correct the spelling, please tell me if there needs further modifications. 

As for the performance, I actually observed no obvious improvements on our own datasets with `brightness=0.4, contrast=0.4, saturation=0.4, hue=0.3`. The random seed is not manually assigned and the performance is on par with the non-color-aug version.

Reply:
By the way, do you think we should add a `random_seed` field in the configuration file for convenience of assigning the seed for random operations?

Reply:
Having a seed would be useful I think, even though not everything would be deterministic.